### PR TITLE
[KIECLOUD-508] - Prepare RHDM and RHPAM 7.10.x branches for 7.10.1 re…

### PR DESCRIPTION
**Pull request by spolti**
…lease
See: https://issues.redhat.com/browse/KIECLOUD-508

Signed-off-by: spolti <fspolti@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Review by msmagnanijr — DISMISSED** [review]
lgtm!

File: quickstarts/library-process/pom.xml [hunk 1]
```diff
@@ -28,8 +28,8 @@
 			resources, i.e. build is platform dependent! -->
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<maven.compiler.version>3.8.0</maven.compiler.version>
-		<ba-platform-bom.version>7.8.1.redhat-00002</ba-platform-bom.version>
-		<org.jboss.bom.version>7.3.2.GA</org.jboss.bom.version>
+		<ba-platform-bom.version>7.10.1.redhat-00006</ba-platform-bom.version>
```
Comment (AntStephenson) [inline quickstarts/library-process/pom.xml:31]: Same as the other PR, this should be 7.10.1.redhat-00001

**Review by AntStephenson — CHANGES_REQUESTED** [review]

File: quickstarts/library-process/pom.xml (hunk 1 above)
Comment (spolti) [reply quickstarts/library-process/pom.xml]: updated, thanks

**Comment by spolti** [comment]
@mramendi hi, can you please review the doc changes?

**Comment by AntStephenson** [comment]
@spolti can we merge this now and review the docs later so we can proceed with the build. I think we are only waiting for this PR, thanks! :)

**Comment by spolti** [comment]
@AntStephenson sure, can you please approve the changes?

**Review by AntStephenson — APPROVED** [review]

**Pull request merged**